python-package/lightgbm/basic.py

```diff
@@ -2869,7 +2881,7 @@ def set_init_score(
 
         Parameters
         ----------
-        init_score : list, list of lists (for multi-class task), numpy array, pandas Series, pandas DataFrame (for multi-class task), or None
+        init_score : list, list of lists (for multi-class task), numpy array, pandas Series, pandas DataFrame (for multi-class task), pyarrow Array, pyarrow ChunkedArray, pyarrow Table (for multi-class task) or None
```

Comment:
Why is this `(for multi-class task)` qualifier necessary?

Couldn't all of a `pyarrow.Array`, `pyarrow.ChunkedArray`, and `pyarrow.Table` be used to provide a 2-dimensional matrix? Notice that we don't have `(for multi-class task)` in the docstring entry for `numpy array`, for exactly that reason.

Reply:
dismissing this, per https://github.com/microsoft/LightGBM/pull/6167#issuecomment-1823611526